### PR TITLE
New FT recoding edits

### PR DESCRIPTION
**Pull request by sfergusond**
Changed the recoding method to align with the code used for the other variables

### CODE REVIEW

File: Data.Rmd [hunk 1]
```diff
@@ -11,6 +11,7 @@ library(tidyverse)
 library(readstata13)
 library(skimr)
 library(psych)
+library(naniar) # use install.packages('naniar') in the console
```
Comment (rrodriguezbarron) [inline Data.Rmd:14]: @sfergusond what does this library do?

File: Data.Rmd [hunk 2]
```diff
@@ -95,9 +96,9 @@ clean <- raw %>%
   # Recoding radio
   mutate_at(vars(contains("radio_")), funs(if_else(.<0,NA_real_,.))) %>% 
   # Recoding web
-  mutate_at(vars(contains("web_")), funs(if_else(.<0,NA_real_,.)))
-  
-  
+  mutate_at(vars(contains("web_")), funs(if_else(.<0,NA_real_,.))) %>%
+  # Recoding thermometers
+  mutate_at(vars(contains("ft_")), funs(if_else(.<0 | .>100, NA_real_,.)))
```
Comment (rrodriguezbarron) [inline Data.Rmd:101]: Cool. Keeping the same format does help a lot.

File: Data.Rmd (hunk 1 above)
Comment (sfergusond) [reply Data.Rmd:14]: I used it for the old recoding method, not necessary anymore

Comment (rrodriguezbarron) [reply Data.Rmd:14]: OK, will reject this line.

**Pull request closed without merging**